docs/datacite.README.md

```diff
@@ -0,0 +1,121 @@
+# DataCite DOI implementation
+
+## DataCite Documentation and References
+
+- DataCite Rest API Guide: https://support.datacite.org/docs/api
+- DataCite REST API Reference: https://support.datacite.org/reference/introduction
+- DataCite Metadata Schema 4.4: https://schema.datacite.org/meta/kernel-4.4/
+
+## DataCite background info
+
+### DOI states, EZID vs DataCite
+
+- reserved (EZID) <==> draft (DataCite)
+- public (EZID) <==> findable (DataCite)
+- unavailable (EZID) <==> registered (DataCite)
```

Comment:
Given that this code migrates EZID things to DataCite, maybe there isn't any need to mention EZID anywhere in here going forward?

Reply:
All mentions of EZID have been removed from the document.